crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -489,6 +496,66 @@ where
             .await?;
 
         Ok((response, ret_api))
+    } else if let Some(matches) = matches.subcommand_matches("perftest") {
+        let mut ops = matches.value_of("ops").unwrap().parse::<u64>().unwrap();
+
+        info!("Performing {} operations", ops);
+
+        let perftest_ops_api = api.clone();
+
+        let mut ops_copy = ops;
+        let start_time = Instant::now();
+        tokio::task::spawn(async move {
+            while ops > 0 {
+                let identity = AuthId::chronicle();
+                let namespace = system_namespace();
+                let _res = perftest_ops_api.handle_perftest(identity, namespace).await;
+                ops -= 1;
+            }
+            info!("Perftest operations sent");
+        });
+
+        let perftest_notify_api = api.clone();
+
+        let (elapsed_time, failed_submissions) = tokio::task::spawn(async move {
+            let mut failed_submissions = 0;
+            while ops_copy > 0 {
+                let mut tx_notifications = perftest_notify_api.notify_commit.subscribe();
```

Comment:
If I move this two lines above, outside the while loop, performing 2500 transactions completes in 282 seconds, but with 1607 rejections due to queuing errors.